stdlib/src/builtin/reversed.mojo

```diff
@@ -90,7 +90,12 @@ fn reversed[
     Args:
         value: The list to get the reversed iterator of.
 
+    Parameters:
+        mutability: Indicates if the list is mutable or immutable.
+        self_life: The lifetime policy for the list.
+        T: The type of elements contained in the list.
+
     Returns:
-        The reversed iterator of the list.
+        _ListIter: The reversed iterator of the list.
```

Comment:
**Question** Why does this `Returns` need to be "named" with the type of the returned value?

Reply:
Hi @ConnorGray, you are correct, there is no need for defining return type. I've removed the redundant return type name from the documentation string. It now reads:

`Returns: The reversed iterator of the list.`